kong/plugins/prometheus/prometheus.lua

```diff
@@ -494,9 +494,9 @@ end
 -- It will get the metrics from the dictionary, sort them, and expose them
 -- aling with TYPE and HELP comments.
 function Prometheus:collect()
-  ngx.header.content_type = "text/plain"
+  kong.response.set_header("Content-Type", "text/plain")
```

Comment:
This is a vendored dependency (https://github.com/knyar/nginx-lua-prometheus) and should be updated only for additional features.

Let's keep this file as is and let it use OpenResty APIs.